throughput-plotter.py

```diff
@@ -0,0 +1,178 @@
+# %%
+import os
+import matplotlib.pyplot as plt
+import matplotlib
+import pandas as pd
```

Comment:
I realize this is just a helper script, but I think this file could be modularized and cleaned up a bit. there are still some old comments and code that is unreachable that is left in. 

A couple more comments would be nice too.